lib/__tests__/standalone-ext.test.js

```diff
@@ -0,0 +1,125 @@
+	});
+
+	test('default', async () => {
+		const { results } = await standalone({
+			files: [path.join(fixturesPath, 'extensions')],
+			config: { rules: { 'color-named': 'never' } },
+		});
+
+		expect(results).toHaveLength(1);
+		expect(results[0].source.endsWith(path.join('extensions', 'folder', 'qux.css'))).toBe(true);
+	});
+
+	test('dot prefix', async () => {
+		const { results } = await standalone({
+			files: [path.join(fixturesPath, 'extensions')],
+			config: { rules: { 'color-named': 'never' } },
+			extensions: ['.scss'],
+			customSyntax: 'postcss-scss',
+		});
+
+		expect(results).toHaveLength(1);
+		expect(results[0].source).toContain('bar.scss');
+	});
+});
+
+describe('inputs', () => {
+	const cwd = path.join(__dirname, 'fixtures', 'extensions');
+
+	safeChdir(cwd);
+
+	test('.', async () => {
+		const { results } = await standalone({
+			files: ['.'],
+			config: configColorNamed,
+			extensions: ['css', 'scss'],
+			customSyntax: 'postcss-scss',
+			cwd,
+		});
+
+		expect(results).toHaveLength(2);
+		expect(existsSync('foo.json')).toBe(true);
+		expect(existsSync('folder/baz.json')).toBe(true);
+		expect(results[0].source).toContain('bar.scss');
+		expect(results[1].source).toContain(path.normalize('folder/qux.css'));
+	});
+
+	test('false positive', async () => {
+		const inputFiles = [cwd, path.join(fixturesPath, 'config-color-named.json')];
+		const { results } = await standalone({
+			files: inputFiles,
+			config: configColorNamed,
+			extensions: ['css', 'scss'],
+			customSyntax: 'postcss-scss',
+			cwd,
+		});
+
+		expect(results).toHaveLength(3);
+		expect(results[0].source).toContain('bar.scss');
+		expect(results[1].source).toContain(path.normalize('folder/qux.css'));
+		expect(results[2].source).toContain(path.normalize('fixtures/config-color-named.json'));
+	});
+
+	test('common prefix', async () => {
+		const warn = jest.spyOn(console, 'warn').mockImplementation(() => {});
+		const inputFiles = [path.join(cwd, 'folder'), path.join(cwd, 'folder-slash', 'file.json')];
+		const { results } = await standalone({
+			files: inputFiles,
+			config: configColorNamed,
+			extensions: ['css'],
+			cwd,
+		});
+
+		expect(results).toHaveLength(2);
+		expect(results[0].source).toContain(path.normalize('folder/qux.css'));
+		expect(results[1].source).toContain(path.normalize('folder-slash/file.json'));
+		warn.mockRestore();
+	});
+
+	test('**', async () => {
+		const { results } = await standalone({
+			files: ['**'],
+			config: configColorNamed,
+			extensions: ['css', 'scss'],
+			customSyntax: 'postcss-scss',
+			cwd,
+		});
+
+		expect(results).toHaveLength(5);
+		expect(results[0].source).toContain('bar.scss');
+		expect(results[1].source).toContain('foo.json');
+		expect(results[2].source).toContain(path.normalize('folder/baz.json'));
+		expect(results[3].source).toContain(path.normalize('folder/qux.css'));
+		expect(results[4].source).toContain(path.normalize('folder-slash/file.json'));
+	});
```

Comment:
[suggest] How about adding a case for no match like this?

```js
test('no match', () => {
	return expect(
		standalone({
			files: ['.'],
			config: configColorNamed,
			extensions: ['js'],
			cwd,
		}),
	).rejects.toThrow('All input files were ignored');
});
```

Reply:
It unearthed a bug, thanks.